spec/policies/form_policy_spec.rb

```diff
@@ -32,10 +32,41 @@
         end
       end
     end
+
+    context "with a trial role" do
+      context "when the user created the form" do
+        context "with an organisation" do
+          let(:user) { build :user, :with_trial, organisation_slug: "gds" }
+
+          it { is_expected.to permit_actions(%i[can_view_form]) }
+        end
+
+        context "without an organisation" do
+          let(:form) { build :form, creator_id: 1234 }
+          let(:user) { build :user, :with_no_org, :with_trial, id: 1234 }
+
+          it { is_expected.to permit_actions(%i[can_view_form]) }
+        end
+      end
+
+      context "when the user didn't create the form" do
+        context "with a different user" do
+          let(:user) { build :user, id: 321 }
+
+          it { is_expected.to forbid_actions(%i[can_view_form]) }
+        end
+
+        context "without a form creator" do
+          let(:form) { build :form }
+
+          it { is_expected.to forbid_actions(%i[can_view_form]) }
+        end
+      end
+    end
   end
 
   describe "#can_add_page_routing_conditions?" do
-    let(:form) { build :form, pages:, org: "gds" }
+    let(:form) { build :form, pages:, org: "gds", creator_id: 123 }
```

Comment:
Is adding a `creator_id` to this test form needed for some reason? I'm surprised that the `#can_add_page_routing_conditions?` check is affected by the creator ID.